README.md

```diff
@@ -1,6 +1,33 @@
-# New Acropolis UK API  [![Build Status](https://travis-ci.org/NewAcropolis/api.svg?branch=master)](https://travis-ci.org/NewAcropolis/api) 
+# New Acropolis UK API  [![Build Status](https://travis-ci.org/NewAcropolis/api.svg?branch=master)](https://travis-ci.org/NewAcropolis/api)
 [![Coverage Status](https://coveralls.io/repos/github/NewAcropolis/api/badge.svg?branch=master)](https://coveralls.io/github/NewAcropolis/api?branch=master)
 
+## Create virtualenv (optional)
```

Comment:
I would say that it can be optional but good practice to create the virtualenv so probably remove the optional

Reply:
Removed the optional